specification/azuredata/resource-manager/Microsoft.AzureData/preview/2019-07-24-preview/azuredata.json

```diff
@@ -2240,10 +2290,11 @@
       }
     },
     "DataControllerProperties": {
+      "description": "The data controller properties.",
       "type": "object",
       "properties": {
-        "dataController": {
-          "$ref": "#/definitions/DataController"
+        "onPremiseProperty": {
+          "$ref": "#/definitions/OnPremiseProperty"
         },
```

Comment:
Is this change relevant with this change on adding of systemData? If not, I suggest to revert this and maybe open another PR specified with replacing this attribute

Reply:
Hi @ArcturusZhang , this is not relevant to adding systemData. I removed this from this PR and will send out a different one for it, as you suggested.

Reply:
And a heads-up that that change would cause breaking changes warnings